src/network/websockets/server/index.ts

```diff
@@ -53,6 +53,25 @@ export function runWebsocketServer<AuthDetails>(params: Params<AuthDetails>) {
       socket.send(JSON.stringify(obj));
     };
 
+    let cachedMessages: Array<string> = [];
+
+    const processCache = () => {
+      if (cachedMessages.length > 0) {
```

Comment:
Conditional before loop processing is meaningless – if the list is empty, the loop won’t do anything anyway. Combine this with `forEach` and the following is much more concise:

```ts
// Before
const processCache = () => {
  if (cachedMessages.length > 0) {
    for (var msg of cachedMessages) {
      handleMsg(msg);
    }
    cachedMessages = [];
  }
};

// After
const processCache = () => {
  cachedMessages.forEach(handleMsg);
  cachedMessages = [];
};
```